src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/PostSetup.java

```diff
@@ -206,6 +208,34 @@ private static void loadSmelteryRecipes() {
         }
     }
 
+    private static void loadJuicer() {
+        List<ItemStack[]> juicerRecipes = new ArrayList<>();
+
+        Juicer juicer = (Juicer) SlimefunItems.JUICER.getItem();
+        if (juicer == null) {
+            return;
+        }
+
+        ItemStack[] input = null;
+
+        for (ItemStack[] recipe : juicer.getRecipes()) {
+
+            if (input == null) {
+                input = recipe;
+                continue;
+            }
+
+            if (input[0] != null && recipe[0] != null) {
+                juicerRecipes.add(new ItemStack[] { input[0], recipe[0] });
+            }
+
+            input = null;
+        }
+
+        Stream<ItemStack[]> stream = juicerRecipes.stream();
+        stream.forEach(recipe -> registerMachineRecipe("ELECTRIC_JUICER", 2, new ItemStack[] { recipe[0] }, new ItemStack[] { recipe[1] }));
+    }
```

Comment:
Not a huge fan with how this works but ill wait to see others opinions

Reply:
The only way to load multiblocks recipes into electric machines that I know of is this